<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menu{
	background-color: rgb(255, 255, 255);
}
#widget{
background-color: QLinearGradient(x1: 0,y1: 0,x2: 0,y2: 1,stop: 0 #e66465, stop:1 #9198e5);
border-radius:20px;

}
QPushButton{
	background-color: rgb(217, 217, 217);
border-radius:20px;
	font: 14pt  &quot;MV Boli&quot;;
}

#menu_2{
	background-color: rgb(255, 255, 255);
	border-radius:20px;
}
QLabel
{
font: 14pt  &quot;MV Boli&quot;;
}
#widget_3
{
	 border-radius:20px;
}
QPushButton::pressed{

	background-color: rgb(125, 125, 125);

}</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>671</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0)
</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 0);

font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>    Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>50</y>
     <width>1351</width>
     <height>771</height>
    </rect>
   </property>
   <widget class="QWidget" name="menu_2" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>611</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>440</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>gestion des Elèves</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion d'Enseignants</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>331</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>gestion des Activités</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
